### ISSUE TITLE
Automatically trigger a W9 collection email when someone registers

### ISSUE THREAD

**Issue by markmhx** [blockstack/app.co#257]

**Comment by markmhx** [comment]
@hstove Is this an idea that's still relevant to our approach to collecting W9 information?

**Comment by hstove** [comment]
Yeah I think so, not high priority but nice to have. Although it'll definitely be a part of our "new" registration flow, whenever that happens. Not as an email, but likely a form you simply upload on app.co.
